### PR TITLE
Another 10 MH3 cards

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+SVar:DBImpulseDraw:DB$ Dig | Cost$ Discard<1/Card> | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | SubAbility$ DBEffect | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ You may discard a card. If you do, exile the top two cards of your library. You may play them until the end of your next turn.
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ UntilTheEndOfYourNextTurn
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play them until the end of your next turn.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ spellgorger_weird | TokenOwner$ You | SpellDescription$ Create a Spellgorger Weird token. (It's a {2}{R} 2/2 Weird creature with "Whenever you cast a noncreature spell, put a +1/+1 counter on Spellgorger Weird.")
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt:10]: This one needs the Token-Script from the other MR

File: forge-gui/res/cardsfolder/upcoming/wurmcoil_larva.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Wurmcoil Larva
+ManaCost:3 B B
+Types:Artifact Creature Phyrexian Wurm
+PT:3/3
+K:Deathtouch
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME dies, create a 1/2 black Phyrexian Wurm artifact creature token with deathtouch and a 2/1 black Phyrexian Wurm artifact creature token with lifelink.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ b_1_2_a_phyrexian_wurm_deathtouch,b_2_1_a_phyrexian_wurm_lifelink
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/wurmcoil_larva.txt:8]: This one needs Token-Scripts from the other MR

File: forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt:10]: indeed, I think we can just keep this open until then
I've messed with enough commits for today :)

File: forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Name:Ral and the Implicit Maze
+ManaCost:3 R R
+Types:Enchantment Saga
+K:Chapter:3:DBDamageAll,DBImpulseDraw,DBToken
+SVar:DBDamageAll:DB$ DamageAll | ValidCards$ Creature.OppCtrl,Planeswalker.OppCtrl | NumDmg$ 2 | SpellDescription$ CARDNAME deals 2 damage to each creature and planeswalker your opponents control.
+SVar:DBImpulseDraw:DB$ Dig | Cost$ Discard<1/Card> | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | SubAbility$ DBEffect | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ You may discard a card. If you do, exile the top two cards of your library. You may play them until the end of your next turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt:6]: needs AB for cost

File: forge-gui/res/cardsfolder/upcoming/refurbished_familiar.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Refurbished Familiar
+ManaCost:3 B
+Types:Artifact Creature Zombie Rat
+PT:2/1
+K:Affinity:Artifact
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDiscard | TriggerDescription$ When CARDNAME enters the battlefield, enters the battlefield, each opponent discards a card. For each opponent who can't, you draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/refurbished_familiar.txt:7]: typo

File: forge-gui/res/cardsfolder/upcoming/retrofitted_transmogrant.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Retrofitted Transmogrant
+ManaCost:B
+Types:Artifact Creature Zombie
+PT:1/1
+A:AB$ ChangeZone | Cost$ 3 B | Origin$ Graveyard | Destination$ Battlefield | Tapped$ True | WithCountersType$ P1P1 | WithCountersAmount$ 2 | ActivationZone$ Graveyard | SpellDescription$ Return CARDNAME from your graveyard to the battlefield tapped.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/retrofitted_transmogrant.txt:5]: desc incomplete

**Review by Hanmac — APPROVED** [review]

**Pull request merged**